src/components/HomePage/styles/Link.css

```diff
@@ -14,6 +14,21 @@
 	margin: 0 auto;
 }
 
+.contactBtn {
+    background: transparent;
+    border: none;
+    outline: none;
+    display: inline-block;
+}
+
+.contactBtn:hover {
+    text-decoration: underline;
+}
+
+.contactBtn:active, .contactBtn:focus {
+    outline: none;
+}
+
```

Comment:
Hmm. I'm a bit confused. Where do these styles get used?

Reply:
Meant to remove! My b